phoenix-core/src/it/java/org/apache/phoenix/end2end/QueryMoreIT.java

```diff
@@ -511,4 +513,122 @@ private void upsertRows(PhoenixConnection conn, String fullTableName) throws SQL
             stmt.execute();
         }
     }
+
+    @Test public void testRVCWithDescAndAscendingPK() throws Exception {
+        final Connection conn = DriverManager.getConnection(getUrl());
+        String fullTableName = generateUniqueName();
+        try (Statement stmt = conn.createStatement()) {
+            stmt.execute("CREATE TABLE " + fullTableName + "(\n"
+                    + "    ORGANIZATION_ID CHAR(15) NOT NULL,\n" + "    SCORE VARCHAR NOT NULL,\n"
```

Comment:
nit: return and have a new line after you have "\n". VIsually it's gonna be easier to read.

Reply:
Sure